test/testaudiostreamdynamicresample.c

```diff
@@ -140,20 +397,14 @@ int main(int argc, char *argv[])
         i += consumed;
     }
 
-    slider_area.x = state->window_w * (1.f - SLIDER_WIDTH_PERC) / 2;
-    slider_area.y = state->window_h * (1.f - SLIDER_HEIGHT_PERC) / 2;
-    slider_area.w = SLIDER_WIDTH_PERC * state->window_w;
-    slider_area.h = SLIDER_HEIGHT_PERC * state->window_h;
-
-    slider_fill_area = slider_area;
-    slider_fill_area.w /= 2;
-
     /* Load the SDL library */
     if (!SDLTest_CommonInit(state)) {
         SDL_LogError(SDL_LOG_CATEGORY_APPLICATION, "Couldn't initialize SDL: %s\n", SDL_GetError());
         return 1;
     }
 
+    FONT_CHARACTER_SIZE = 16;
```

Comment:
I don't love this being a global variable in the test utilities, but let's change this after this PR merges.